ledger/metrics/src/main/scala/com/daml/metrics/api/opentelemetry/OpenTelemetryFactory.scala

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics.api.opentelemetry
+
+import java.time.Duration
+import java.util.concurrent.TimeUnit
+
+import com.daml.metrics.api.Gauges.VarGauge
+import com.daml.metrics.api.MetricHandle.Timer.TimerHandle
+import com.daml.metrics.api.MetricHandle.{Counter, Factory, Gauge, Histogram, Meter, Timer}
+import com.daml.metrics.api.{MetricHandle, MetricName}
+import io.opentelemetry.api.metrics.{
+  LongCounter,
+  LongHistogram,
+  LongUpDownCounter,
+  Meter => OtelMeter,
+}
+
+trait OpenTelemetryFactory extends Factory {
+
+  def otelMeter: OtelMeter
```

Comment:
Nit: can we add newlines between the non-one-liner method definitions, please? It's hard to read the code without the previous.